ci/examples/blk.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+# Copyright 2025, UNSW
+# SPDX-License-Identifier: BSD-2-Clause
+
+import asyncio
+import functools
+import subprocess
+from pathlib import Path
+from tempfile import TemporaryDirectory
+import sys
+import tempfile
+
+sys.path.insert(1, Path(__file__).parents[2].as_posix())
+
+from ci.hardware_backend import *
+from ci.runner import TestConfig, cli, matrix_product
+
+TEST_MATRIX = matrix_product(
+    board=(
+        "maaxboard",
+        "qemu_virt_aarch64",
+        "qemu_virt_riscv64",
+    ),
+    # only prints output in debug mode
+    config=("debug",),
```

Comment:
```suggestion
    config=("debug",),
```

Hmm yes. I guess we should fix the block example to output in release mode as well.

Reply:
I can try do that now.